android/app/src/main/java/com/reactnativenavigation/activities/BaseReactActivity.java

```diff
@@ -252,6 +252,16 @@ public Screen popToRoot(String navigatorId) {
         return null;
     }
 
+    @CallSuper
+    public Screen resetTo(Screen screen) {
+        setNavigationStyle(screen);
+        if (mToolbar != null) {
+            mToolbar.hideBackButton();
```

Comment:
Do we need to hide the back button when calling `resetTo`? Maybe this is a copy-paste error from `popToRoot`?